quinn-proto/src/varint.rs

```diff
@@ -60,6 +62,18 @@ impl VarInt {
             unreachable!("malformed VarInt");
         }
     }
+
+    /// Length of an encoded value from its first byte
+    pub fn size_encoded(first: u8) -> usize {
+        let tag = first >> 6;
+        match tag {
+            0b00 => 1,
```

Comment:
This could be more concisely written as `pow()`. :)

Reply:
haha, oops :sweat_smile: